docs/tutorials/data_visualization.ipynb

```diff
@@ -197,39 +252,92 @@
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "titanic_numerical.plot_boxplots()"
    ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "start_time": "2023-06-17T18:33:29.146080Z",
+     "end_time": "2023-06-17T18:33:30.540435Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's look at the different boxplots of all numerical columns:\n",
+    "\n",
+    "* The 2nd boxplot, right to the `age` boxplot, depicts the number of passengers with `siblings/spouses` aboard the Titanic from 0 to 8. Here, the middle 50% of passengers had 1 sibling/spouse and the rest were considered outliers.\n",
+    "* The `parents/children` boxplot illustrates the count rate of the number of passengers with `parents/children`. More than 80% of the passengers had no parents nor children with them because the upper and lower whiskers including the box are on 0. The rest were considered outliers.\n",
+    "* The `travel_class` boxplot has its middle 50% of passengers either in the 2nd or 3rd class. The remaining passengers were considered first class.\n",
+    "* In the `fare` boxplot, the box lies somewhere between 10 and 40. This implies that about half of the passengers paid a passenger fare below 20. The other half paid above 20. It was also dominated by extreme outliers. An extreme outlier is a value 3 times the difference between the 75th percentile and the 25th percentile (3 x IQR).\n",
+    "* The last boxplot, `survived`, depicts the survival rate of the middle 50% of passengers between 0.0 and 1.0."
```

Comment:
This doesn't seem to be relevant information. Maybe add  something along the lines of "This boxplot is not really helpful because obviously a passenger either survived or not"

Reply:
It is actually. It demonstrates a bad example of using a boxplot.